modules/op-creating-and-verifying-task-run-signatures-without-any-additional-authentication.adoc

```diff
@@ -47,7 +47,7 @@ $ oc patch configmap chains-config -n openshift-pipelines -p='{"data":{"artifact
 $ oc delete po -n openshift-pipelines -l app=tekton-chains-controller
 ----
 
-. Create a task run. 
+. Create a task run.
 +
 [source,terminal]
 ----
```

Comment:
Are we allowed to reference arbitrary yaml files from Github?

Reply:
@ppitonak could you please clarify your request? I don't see what would disallow it?

Reply:
In the past when we were creating the doc using the Tutorial, we were said that Tutorial source code has to be in a supportable GitHub org such as `openshift` if we want to reference yaml files. :shrug: